designsafe/apps/auth/views.py

```diff
@@ -198,3 +203,9 @@ def agave_oauth_callback(request):
 
 def agave_session_error(request):
     return render(request, 'designsafe/apps/auth/agave_session_error.html')
+
+# load_alerts_into_session() will be called after a new login ( within agave_oauth_callback() )
+def load_alerts_into_session(request):
+    alert_messages = [model_to_dict(alert) for alert in PageAlert.objects.all()]
```

Comment:
Doesn't `model_to_dict` also adds the model type and id to the returned object?
Do we need that extra info there?
Maybe we could just do something more granular like:
```suggestion
    alert_messages = [{"alert_type": alert.alert_type, "alert_message": alert.alert_message} for alert in PageAlert.objects.all()]
```